tools/downloader/topologies.py

```diff
@@ -0,0 +1,99 @@
+import os
+import argparse
+
+from pathlib import Path
+
+import common
+import downloader
+import converter
+
+class Topology:     
+    '''
+    {}
+    Model task type: {}
+
+    License: {}
+    '''
+
+    def __init__(self, model_name, *, precision=None, download_dir=None):
+        '''
+        Downloads target model. If the precision doesn't set, loads all the available models.
+        If there is already downloaded model - retrieves the model from cache.
+        Creates an object which stores information about the downloaded model.
+
+        Parameters
+        ----------
+            model_name
+                Target model name.
+            precision
+                Target model precision. By default is None.
+            download_dir
+                Model download directory.
+        '''
+        if not download_dir: 
+            download_dir = Path(Path('.').resolve() / 'models')
+        cache_dir = Path(Path(download_dir) / '.cache')
```

Comment:
```suggestion
        cache_dir = download_dir / '.cache'
```

Reply:
But then we can't use it like this:
```python
ssd = omz.Model('mobilenet-ssd', download_dir='/user/model')
```
We should do ```download_dir=Path('/user/model')```
Is it ok?

Reply:
Yeah, fair enough. I think the version you have now is the best.

Reply:
Why not to have 
```python
download_dir = Path(download_dir)
```

at the beginning of `__init__`? So 
```
omz.Model('modelname', download_dir='/path/to/model') 
```
will be also valid